includes/definitions/ios.yaml

```diff
@@ -8,6 +8,15 @@ over:
     - { graph: device_processor, text: 'CPU Usage' }
     - { graph: device_mempool, text: 'Memory Usage' }
 icon: cisco
+discovery:
+    -
+      sysDescr:
+          - 'Cisco Internetwork Operating System Software'
+          - 'IOS (tm)'
+          - 'Cisco IOS Software'
+          - 'Global Site Selector'
+      sysDescr_except:
+          - 'LINUX_IOSD'
```

Comment:
Sure, I had removed it earlier as I couldn't find any devices containing IOS-XE.  As I said on the other PR, no harm in adding it.